src/effects/effectsmanager.cpp

```diff
@@ -202,6 +215,25 @@ void EffectsManager::readEffectsXml() {
     m_pVisibleEffectsList->readEffectsXml(doc, m_pBackendManager);
 }
 
+void EffectsManager::readEffectsXmlSingleDeck(const QString& deckGroup) {
+    QDir settingsPath(m_pConfig->getSettingsPath());
```

Comment:
Normally we prefer initialization with = `QDir settingsPath = m_pConfig->getSettingsPath();`
To have a clear visible distinguishing from a function call. 

Recently we have discussed to prefer initialization lists like:  
`QDir settingsPath{m_pConfig->getSettingsPath()};`

This is just a hint, because that never made the way into our style guides.

Reply:
ackknowldeged.
It's copy/paste from `readEffectsXml()`, and actually it's mixed in the vaious effect managers already:
QString xyPath(m_pConfig->getSettingsPath() + ...);
QDir xyDir(xyPath);
vs
QFile xyFile(m_pConfig->getSettingsPath() + ...);